### ISSUE TITLE
naml codify makefile

### ISSUE THREAD

**Issue by kris-nova** [kris-nova/naml#65]
lets auto generate a makefile as well for our friends

maybe syntax like this

```bash
kubectl get deploy -oyaml | naml codify --name=beeps > main.go
naml codify makefile --name=beeps > Makefile
make
./beeps
sudo make install
beeps --help
```

**Comment by kris-nova** [comment]
naml outputs code so it can be used with the `>` operator in linux

we can add another subcommand to autogenerate a makefile

**Comment by kris-nova** [comment]
closed in favor of the #108
